app/components/TrustedSourceList.js

```diff
@@ -0,0 +1,88 @@
+import findIndex from 'lodash/findIndex';
+import React from 'react';
+import { StyleSheet, Switch, TouchableOpacity } from 'react-native';
+import { MenuOption } from 'react-native-popup-menu';
+
+import { Typography } from '../components/Typography';
+import Colors from '../constants/colors';
+import languages from '../locales/languages';
+
+export const TrustedSourceList = ({
+  selectedAuthorities,
+  authoritiesList,
+  isAuthorityFilterActive,
+  toggleFilterAuthoritesByGPSHistory,
+  addAuthorityToState,
+  filterAuthoritesByGPSHistory,
+  onMenuClick,
+}) => {
+  const isAuthoritySelected = name => {
+    return findIndex(selectedAuthorities, ['key', name]) < 0;
```

Comment:
```suggestion
    return findIndex(selectedAuthorities, {key: name}) < 0;
```

This is a little more clear

Reply:
I realise due to the format of the authorities list, this will not work :) never mind.